Parsers/ASimProcessEvent/Parsers/ASimProcessCreateSentinelOne.yaml

```diff
@@ -0,0 +1,108 @@
+ParserName: ASimProcessCreateSentinelOne
+EquivalentBuiltInParser: _ASim_ProcessEvent_CreateSentinelOne
+ParserParams:
+  - Name: disabled
+    Type: bool
+    Default: false
+ParserQuery: |
+  let parser = (disabled: bool=false) {
+      SentinelOne_CL
+      | where not(disabled) 
+          and event_name_s == "Alerts."
+          and alertInfo_eventType_s == "PROCESSCREATION" 
+      | project-rename
+          DvcId = agentDetectionInfo_uuid_g,
+          EventStartTime = sourceProcessInfo_pidStarttime_t,
+          TargetProcessCommandLine = targetProcessInfo_tgtProcCmdLine_s,
+          TargetProcessId = targetProcessInfo_tgtProcPid_s,
+          TargetProcessName = targetProcessInfo_tgtProcName_s,
+          EventUid = _ItemId,
+          TargetProcessCreationTime = targetProcessInfo_tgtProcessStartTime_t,
+          ActingProcessName = sourceProcessInfo_name_s,
+          ParentProcessName = sourceParentProcessInfo_name_s,
+          ActingProcessCommandLine = sourceProcessInfo_commandline_s,
+          ActingProcessGuid = sourceProcessInfo_uniqueId_g,
+          ActingProcessSHA1 = sourceProcessInfo_fileHashSha1_s,
+          ParentProcessSHA1 = sourceParentProcessInfo_fileHashSha1_s,
+          ActingProcessSHA256 = sourceProcessInfo_fileHashSha256_s,
+          ParentProcessSHA256 = sourceParentProcessInfo_fileHashSha256_s,
+          DvcOs = agentDetectionInfo_osName_s,
+          DvcOsVersion = agentDetectionInfo_osRevision_s,
+          TargetProcessIntegrityLevel = targetProcessInfo_tgtProcIntegrityLevel_s,
+          EventOriginalType = alertInfo_eventType_s,
+          EventOriginalSeverity = ruleInfo_severity_s,
+          EventOriginalUid = alertInfo_dvEventId_s
+      | invoke _ASIM_ResolveDvcFQDN('agentDetectionInfo_name_s')
+      | extend
+          ActingProcessId = sourceProcessInfo_pid_s,
+          ActorUsername = sourceProcessInfo_user_s,
+          TargetUsername = sourceProcessInfo_user_s,
+          Hash = coalesce(targetProcessInfo_tgtFileHashSha256_s, targetProcessInfo_tgtFileHashSha1_s),
+          ParentProcessId = sourceProcessInfo_pid_s,
+          TargetProcessSHA1 = targetProcessInfo_tgtFileHashSha1_s,
+          TargetProcessSHA256 = targetProcessInfo_tgtFileHashSha256_s,
+          ParentProcessMD5 = replace_string(sourceParentProcessInfo_fileHashMd5_g, "-", ""),
+          ActingProcessMD5 = replace_string(sourceProcessInfo_fileHashMd5_g, "-", ""),
+          EventSeverity = iff(EventOriginalSeverity  == "Critical", "High", EventOriginalSeverity)
+      | extend
+          EventCount = int(1),
+          EventProduct = "SentinelOne",
+          EventResult = "Success",
+          DvcAction = "Allowed",
+          EventSchemaVersion = "0.1.4",
+          EventType = "ProcessCreated",
+          EventVendor = "SentinelOne",
```

Comment:
Please add SentinelOne in EventVendor EventProduct supported list
https://github.com/Azure/Azure-Sentinel/blob/master/ASIM/dev/ASimTester/ASimTester.csv